workspace_analyzer.py

```diff
@@ -943,14 +963,24 @@ def analyze_manipulability_on_object_surface(self,
             return results, stl_mesh
 
         #Step Three: Locate Instances Where the Arm Collides WIth The Object In Question
-        nv = len(stl_mesh.vectors)
+        #nv = len(stl_mesh.vectors)
+        tri_vecs = None
+        #vec_len = 0
+        if approximate_bounding:
+            if bound_shape is not None:
+                tri_vecs = np.array(AlphaShape(filtered_points, 1.2).triangles)
+            else:
+                tri_vecs = np.array(AlphaShape(points, 1.2).triangles)
+        else:
+            tri_vecs = stl_mesh.vectors
+        vec_len = len(tri_vecs)
 
         def isodd(x):
             return x % 2 != 0
 
         num_intersections = np.zeros(joint_locs_arr_len)
         if parallel:
-            chunk_sz = len(stl_mesh.vectors) // self.num_procs
+            chunk_sz = vec_len // self.num_procs
```

Comment:
did not know this was integer division

Reply:
I'll leave a comment explaining that I suppose. It's to determine the chunk size of points I can feed to a parallel worker.